app/javascript/widget/components/PreChat/Form.vue

```diff
@@ -118,13 +118,10 @@ export default {
       const isUserEmailAvailable = !!this.currentUser.email;
       const isUserPhoneNumberAvailable = !!this.currentUser.phone_number;
       return this.preChatFields.filter(field => {
-        if (
+        return !(
```

Comment:
While we are at it can we assign `(isUserEmailAvailable && field.name === 'emailAddress') ||
          (isUserPhoneNumberAvailable && field.name === 'phoneNumber')`  to a variable before doing the return? To make it more verbose

Reply:
I don't think that's necessary. The name of the function and the variables in it are already verbose enough.